test/dummy/app/models/article.rb

```diff
@@ -1,2 +1,11 @@
 class Article < ActiveRecord::Base
+  belongs_to :admin_user
+
+  def computed
+    "Not orderable"
+  end
```

Comment:
Added this just to have a column which cannot be ordered by. I check in the tests that this column is not a link.